### PR TITLE
Update deprecation popup

### PR DESCRIPTION
**Pull request by glass-ships**
## Description of work:

Check all that apply:
- [x] added [release notes](https://github.com/neutrons/RefRed/blob/next/docs/release_notes.rst) (if not, provide an explanation in the work description)
- [na] updated documentation
- [x] Source added/refactored
- [na] Added unit tests
- [na] Added integration tests
- [] Verified that tests requiring the /SNS and /HFIR filesystems pass without fail

**References:**
- Links to IBM EWM items: https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=7686
- Links to related issues or pull requests:

# Manual test for the reviewer
([instructions to set up the environment](https://github.com/neutrons/RefRed/blob/next/docs/developer/testing.rst#running-manual-tests-in-a-pull-request))

Start RefRed GUI
```bash
(refred)> PYTHONPATH=$(pwd):$PYTHONPATH ./scripts/start_refred.py
```
Or run tests
```bash
(refred)> pytest test/unit/RefRed/test_main.py
```

# Check list for the reviewer
- [x] [release notes](https://github.com/neutrons/RefRed/blob/next/docs/release_notes.rst) updated, or an explanation is provided as to why release notes are unnecessary
- [x] best software practices
    + [x] clearly named variables (better to be verbose in variable names)
    + [x] code comments explaining the intent of code blocks
- [x] All the tests are passing
- [x] The documentation is up to date
- [x] code comments added when explaining intent

### CODE REVIEW

**Comment by glass-ships** [comment]
Some formatting noise, but relevant lines are [here](https://github.com/neutrons/RefRed/pull/135/files#diff-1f4557f6cb557bc91c445e3b16b0a4d6d7f66056ebbd9eb200a13c376d8798a7L90)

**Comment by glass-ships** [comment]
Codecov failure is a false positive - simply due to line numbers changing - number of tested lines did not change

File: RefRed/configuration/loading_configuration.py [hunk 1]
```diff
@@ -85,22 +86,24 @@ def check_config_file(self, filename):
...
-            warning_msg += "Use the following to start version 4:\n\n"
-            warning_msg += "    refred --v4\n\n"
-            warning_msg += "Please consult with your local contact for advise."
+            warning_msg = (
+                "The reduction parameters you are about to load are from "
+                + "an older version of the reduction.\n\n"
+                + "It is NOT advised to mix reduction versions.\n\n"
+                + "If you need to change your reduction parameters,"
+                + "it is recommended to reprocess all your data with the new version.\n\n"
+                + "Use the following to start version 4:\n\n"
```
Comment (jmborr) [inline RefRed/configuration/loading_configuration.py:97]: Since we're going to remove `refred --v4`,  we should not ask User to use it.

**Review by jmborr — CHANGES_REQUESTED** [review]

File: RefRed/configuration/loading_configuration.py (hunk 1 above)
Comment (glass-ships) [reply RefRed/configuration/loading_configuration.py]: True. removed that line

**Review by jmborr — APPROVED** [review]
looks :ok_hand: , works :ok_hand:

**Pull request merged**